internal/oci/uvm.go

```diff
@@ -87,6 +87,11 @@ const (
 	// files and information needed to install given driver(s). This may include .sys,
 	// .inf, .cer, and/or other files used during standard installation with pnputil.
 	AnnotationAssignedDeviceKernelDrivers = "io.microsoft.assigneddevice.kerneldrivers"
+	// AnnotationHostProcessInheritUser indicates whether to ignore the username passed in to run a host process
+	// container as and instead inherit the user token from the executable that is launching the container process.
+	AnnotationHostProcessInheritUser = "io.microsoft.container.hostprocess-inherit-user"
```

Comment:
With k8s we need to use something like `microsoft.com/hostprocess-inherit-user`.

https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set

Reply:
Same goes for the hostprocess annotation itself also or? I'm confused as how we haven't had to do this for any others

Reply:
We should adopt this pattern for any annotation that needs to be used with upstream containerd or k8s. The code in upstream containerd to pass through annotations to the shim assumes this pattern is used, and it's also important to make sure k8s is happy with it.

Reply:
Gotcha, changed